typescript/submissions.d.ts

```diff
@@ -208,9 +209,6 @@ export interface NewS3SubmissionData {
   }
   definition: Form
   formsAppId: number
-  ipAddress?: string
-  keyId?: string
-  user?: _FormSubmissionMeta['user']
   device?: S3SubmissionDataDevice
   lastElementUpdated?: FormElement
   externalId?: string
```

Comment:
Can you also please move  `formsAppId` and `externalId` to the `S3SubmissionData` type.